test/suite/test_cursor18.py

```diff
@@ -0,0 +1,272 @@
+        ('var', dict(keyformat='r', valueformat='i')),
+        ('fix', dict(keyformat='r', valueformat='8t')),
+    ]
+
+    scenarios = make_scenarios(types)
+
+    def create(self):
+        self.session.create(self.uri, 'key_format={},value_format={}'.format(self.keyformat, self.valueformat))
+    
+    def verify_keys(self, version_cursor, expected_start_ts, expected_start_durable_ts, expected_stop_ts, expected_stop_durable_ts, expected_type, expected_prepare_state, expected_flags, expected_location):
+        [_, start_ts, start_durable_ts, _, stop_ts, stop_durable_ts, type, prepare_state, flags, location] = version_cursor.get_keys()
+        self.assertEquals(start_ts, expected_start_ts)
+        self.assertEquals(start_durable_ts, expected_start_durable_ts)
+        self.assertEquals(stop_ts, expected_stop_ts)
+        self.assertEquals(stop_durable_ts, expected_stop_durable_ts)
+        self.assertEquals(type, expected_type)
+        self.assertEquals(prepare_state, expected_prepare_state)
+        self.assertEquals(flags, expected_flags)
+        self.assertEquals(location, expected_location)
+
+    def test_update_chain_only(self):
+        self.create()
+
+        cursor = self.session.open_cursor(self.uri, None)
+        # Add a value to the update chain
+        self.session.bigin_transaction()
+        cursor[0] = 0
+        self.session.commit_transaction("commit_timestamp=" + self.timestamp_str(1))
+
+        # Update the value
+        self.session.bigin_transaction()
+        cursor[0] = 1
+        self.session.commit_transaction("commit_timestamp=" + self.timestamp_str(5))
+
+        # Open a version cursor
+        version_cursor = self.session.open_cursor(self.uri, "debug=(version_cursor=true)")
+        version_cursor.set_key(0)
+        self.assertEquals(version_cursor.search(), 0)
+        self.assertEquals(version_cursor.next(), 0)
+        self.verify_keys(version_cursor, 5, 5, WT_TS_MAX, WT_TS_MAX, 3, 0, 0, 0)
+        self.assertEquals(version_cursor.get_value(), 1)
+        self.assertEquals(version_cursor.next(), 0)
+        self.verify_keys(version_cursor, 1, 1, 5, 5, 3, 0, 0, 0)
+        self.assertEquals(version_cursor.get_value(), 0)
+        self.assertEquals(version_cursor.next(), wiredtiger.WT_NOTFOUND)
+
+    def test_ondisk_only(self):
+        self.create()
+
+        cursor = self.session.open_cursor(self.uri, None)
+        # Add a value to the update chain
+        self.session.bigin_transaction()
+        cursor[0] = 0
+        self.session.commit_transaction("commit_timestamp=" + self.timestamp_str(1))
+
+        evict_cursor = self.session.open_cursor(self.uri, None, "debug=(release_evict)")
+        self.session.begin_transaction()
+        self.assertEquals(evict_cursor[0], 0)
+        evict_cursor.close()
+        self.session.rollback_transaction()
+
+        # Open a version cursor
+        version_cursor = self.session.open_cursor(self.uri, "debug=(version_cursor=true)")
```

Comment:
I changed the config from `version_cursor` to `dump_version`, so we will need to change this to match.